test/processor/FHIRProcessor.test.ts

```diff
@@ -15,47 +16,53 @@ describe('FHIRProcessor', () => {
   let valueSetSpy: jest.SpyInstance;
 
   beforeAll(() => {
-    processor = new FHIRProcessor(new fhirdefs.FHIRDefinitions());
     structureDefinitionSpy = jest.spyOn(StructureDefinitionProcessor, 'process');
     codeSystemSpy = jest.spyOn(CodeSystemProcessor, 'process');
     valueSetSpy = jest.spyOn(ValueSetProcessor, 'process');
   });
 
   beforeEach(() => {
+    processor = new FHIRProcessor(new fhirdefs.FHIRDefinitions());
     structureDefinitionSpy.mockClear();
     codeSystemSpy.mockClear();
     valueSetSpy.mockClear();
   });
 
   it('should try to process a Profile with the StructureDefinitionProcessor', () => {
-    processor.process(path.join(__dirname, 'fixtures', 'simple-profile.json'));
+    processor.register(path.join(__dirname, 'fixtures', 'simple-profile.json'));
+    processor.process(new Package());
```

Comment:
I don't remember the big picture, but are there use cases where we definitely want to pass in the `Package` to be mutated (which is what I assume it is doing)?  Versus having `process` be a no-arg function and returning a `Package` it constructs itself?

Reply:
Also, if we expect many more tests in this file, we may want to consider creating a helper function (e.g., `registerAndProcess`) so we don't need to make two very similar function calls in every test.